docs/openj9_support.md

```diff
@@ -44,13 +44,12 @@ columns will be removed over time.
-| 0.50.0         | Mar 2025 (2)   |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>   |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |    :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>       |     :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>              |    :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>             |   :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>        |
-| 0.51.0         | May 2025 (1)   |   :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>   |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |    :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>       |     :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>              |    :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>             |   :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>       |
+| 0.51.0         | May 2025   |   :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>   |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |    :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>       |     :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>              |    :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>             |          |
+| 0.52.0         | Jun 2025 (1, 2)   |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>   |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |    :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>       |     :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>              |    :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>             |   :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>        |
```

Comment:
Pls remove 0.52, we are going to skip this release too.

Reply:
@pshipton - So can I remove the JDK24 column as well?